api/lib/resolvers.js

```diff
@@ -340,16 +340,10 @@ const resolvers = {
     networks: (_, { experimental }) => {
       const networks = networkList
         .map((network) => {
-          if (network.id === 'local-cosmos-hub-testnet') {
-            // HACK: network.api_url for the testnet has to be different for internal
-            // (docker DNS to access the testnet container) and external (this frontend to
-            // access the docker container from the outside via its port)
-            return {
-              ...network,
-              api_url: 'http://localhost:9071'
-            }
-          }
-          return network
+          return Object.assign({}, network, {
+            rpc_url: network.public_rpc_url,
+            public_rpc_url: undefined
```

Comment:
Why this? This is what is making get `null` in the FE for `public_rpc_url`. What was your idea here?

Reply:
the idea is to only output `rpc_url` which is the public url. the frontend doesn't need to know about the private url

Reply:
need to adjust in polkadot-transactions line 32

Reply:
ok I see

Reply:
Shouldn't we do the same for the single network query?